docs/aws-sts-provisioning.md

```diff
@@ -67,7 +67,7 @@ May also soon be automated by ccoctl.
 Create a ClusterDeployment normally with the following changes:
 
   1. Create a Secret for your private service account signing key created with ccoctl key-pair above: `kubectl create secret generic bound-service-account-signing-key --from-file=bound-service-account-signing-key.key=serviceaccount-signer.private`
-  1. Create a ConfigMap for your installer manifets (credential role Secrets, Authentication config): `kubectl create configmap cluster-manifests --from-file=manifests/`
+  1. Create a Secret for your installer manifets (credential role Secrets, Authentication config): `kubectl create secret generic cluster-manifests --from-file=manifests/`
```

Comment:
Typo in the original doc.

```suggestion
  1. Create a Secret for your installer manifests (credential role Secrets, Authentication config): `kubectl create secret generic cluster-manifests --from-file=manifests/`
```

Reply:
Good eye. Fixed (here and one other spot).